Content.Server/Nutrition/EntitySystems/CreamPieSystem.cs

```diff
@@ -89,6 +92,11 @@ protected override void CreamedEntity(EntityUid uid, CreamPiedComponent creamPie
                 otherPlayers.RemovePlayer(actor.PlayerSession);
             }
             _popup.PopupEntity(Loc.GetString("cream-pied-component-on-hit-by-message-others", ("owner", uid), ("thrower", args.Thrown)), uid, otherPlayers, false);
+
+            // inject the creampie payload (for silly reasons)
+            // i'd do this with SolutionInjectOnCollide but creampiesystem eats the pie before it can inject so uhhhhhhhhhhhhh
+            if (_solutions.TryGetInjectableSolution(args.Thrown, out _, out var injectable) && TryComp<BloodstreamComponent>(args.Target, out var bloodstream))
+                _bloodstream.TryAddToChemicals(args.Target, injectable, bloodstream);
         }
```

Comment:
We've had this before but it's too overpowered so not gonna add it back in.

Reply:
As it stands, the InjectableSolution isn't modifiable by players in any way so it can't be replaced with something even more weird/dangerous. However, this currently doesn't stop someone from stacking multiple mimana pie creamings to inject ludicrous amounts of the mute toxin.

Would it be better to set the mute as a temporary status effect with a maximum duration not to be exceeded, or is "pie that mutes the victim when thrown" what's on the chopping block?